paper-shadow.html

```diff
@@ -23,34 +23,90 @@
 -->
 
 <link href="../polymer/polymer.html" rel="import">
+<link href="../layout/layout.html" rel="import">
+<dom-module id="paper-shadow">
+  <style>
+    paper-shadow {
+      display: block;
+      position: relative;
+      pointer-events: none;
+    }
+
+    paper-shadow #shadow-bottom,
+    paper-shadow #shadow-top {
+      border-radius: inherit;
+    }
+
+    paper-shadow #shadow-bottom[animated],
+    paper-shadow #shadow-top[animated] {
+      transition: box-shadow 0.28s cubic-bezier(0.4, 0, 0.2, 1);
+    }
+
+    paper-shadow #shadow-top.paper-shadow-z-1 {
+      box-shadow: none;
+    }
+
+    paper-shadow #shadow-bottom.paper-shadow-z-1 {
+      box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.37);
+    }
```

Comment:
I assumed you had a good reason re: using two children to create the shadow. If you don't, I'm totally happy to explore a simpler strategy.

I don't have a major investment in the attributes vs classes fight. I've made a case for classes (see: https://github.com/Polymer/layout/pull/5). I would be very happy to know that we have reached a consensus (or at least an ultimatum) on which strategy to prefer moving forward.